File: LeetCode/cpp/455.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(), g.end());
        sort(s.begin(), s.end());
        int i = s.size() - 1, j = g.size() - 1;
        while (i >= 0 && j >= 0) {
            if(s[i] >= g[j])
                --i;
            --j;
        }
        return s.size() - i - 1;
    }
};

File: LeetCode/python3/455.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        s.sort(reverse=True)
        g.sort(reverse=True)
        i, j = 0, 0
        while i < len(s) and j < len(g):
            if g[j] <= s[i]:
                i += 1
            j += 1
        return i